artifactory_test.go

```diff
@@ -5152,3 +5152,65 @@ func readerCloseAndAssert(t *testing.T, reader *content.ContentReader) {
 func readerGetErrorAndAssert(t *testing.T, reader *content.ContentReader) {
 	assert.NoError(t, reader.GetError(), "Couldn't get reader error")
 }
+
+// Terraform tests : can not be tested yet.
+
+func TestTerraformPublish(t *testing.T) {
+	initArtifactoryTest(t)
+	defer cleanArtifactoryTest()
+	createJfrogHomeConfig(t, true)
+	projectPath := prepareTerraformProject("terraformproject", t, true)
+	log.Info("project path : ", projectPath)
```

Comment:
Let's remove this log line, because I assume we don't want to see it every time the tests run.